ethereum/contracts/SwapFactory.sol

```diff
@@ -91,12 +91,6 @@ contract SwapFactory is Secp256k1 {
         emit Ready(swapID);
     }
 
-    // is_ready returns whether a swap has been set to "ready" or not.
-    // note: it will return false, not revert, if the swap does not exist.
-    function is_ready(bytes32 _swapID) public view returns (bool) {
-        return swaps[_swapID] == Stage.READY;
-    }
```

Comment:
I didn't want to remove this from the contact until we had a reason to regenerate the bindings, which this PR does.